docs/book/src/reference/glossary.md

```diff
@@ -56,6 +56,9 @@ Cluster API Provider IBM Cloud
 ### CAPN
 Cluster API Provider Nested
 
+### CAPX
```

Comment:
Cool acronym! 

We're going to have to make sure to refer all providers as CAP* from now on :slightly_smiling_face:

Reply:
yes i like it too

X is clearly our Nutanix brand ;-)